src/desktop/src/ui/global/FatalError.js

```diff
@@ -33,39 +32,46 @@ const FatalError = ({ error }) => {
             );
         }
 
-        return (
-            <form className={css.tutorial}>
-                <h1>Missing security dependencies</h1>
-                <p>
-                    Your Linux distribution is missing libsecret library that is required for Trinity to securely store
-                    and retrieve sensitive information.
-                </p>
-                <p>Depending on your distribution, you will need to run the following command:</p>
-                <ul>
-                    <li>
-                        Debian/Ubuntu: <pre>sudo apt-get install libsecret-1-dev</pre>
-                    </li>
-                    <li>
-                        Red Hat-based: <pre>sudo yum install libsecret-devel</pre>
-                    </li>
-                    <li>
-                        Arch Linux: <pre>sudo pacman -S libsecret</pre>
-                    </li>
-                </ul>
-                <p>
-                    For some desktop environments (e.g. Kubuntu or KDE), you may need to install an additional library
-                    for the communication with libsecret:
-                </p>
-                <ul>
-                    <li>
-                        <pre>sudo apt-get install gnome-keyring</pre>
-                    </li>
-                </ul>
-            </form>
-        );
-    };
+        if (
+            typeof error === 'string' &&
+            [
+                'The name org.freedesktop.secrets was not provided by any .service files',
+                'Cannot create an item in a locked collection',
+                'Unknown or unsupported transport \'disabled\' for address \'disabled:\'',
```

Comment:
Is it possible to match an error code instead of the localized message? This error may appear as `Unbekannter oder nicht unterstützter Transport »disabled« für Adresse »disabled:«` if the system language is German, for example ([see Bugsnag event](https://app.bugsnag.com/iota-foundation/trinity-desktop/errors/5ce6a1f1dfbdf60019b3a725?filters%5Bevent.since%5D%5B%5D%5Btype%5D=eq&filters%5Bevent.since%5D%5B%5D%5Bvalue%5D=30d&filters%5Berror.status%5D%5B%5D%5Btype%5D=eq&filters%5Berror.status%5D%5B%5D%5Bvalue%5D=open&filters%5Brelease.seen_in%5D%5B%5D%5Btype%5D=eq&filters%5Brelease.seen_in%5D%5B%5D%5Bvalue%5D=latest&event_id=5ce71497003d62b26bf70000))

Reply:
There's no error code available (only a string type error) :/ What we could do is regex match that error to `/disabled(.*)disabled:/g`.

Reply:
Yeah, that would be best I think